### PR TITLE
Add some notes for OS setup

### LINKED ISSUE #323: Add a note in OS setup
I feel that new babies could think they have to install all the software for their OS. Actually, they only need to install the system, and all the other softeware can be installed later when they need to use them. This will save their time to read the tutorials.

### PR DESCRIPTION
**Pull request by core-man**
Fixes #323.

### CODE REVIEW

**Comment by core-man** [comment]
I also add a note for Intel Toolkit since it is not necessary for our research studies.

File: source/computer/setup-note.rst_ [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.. note::
```
Comment (seisman) [inline source/computer/setup-note.rst_:1]: This file is only used twice, perhaps we can just put the content directly in these two files?

Comment (core-man) [reply source/computer/setup-note.rst_]: Done in 579cc5f.

File: source/computer/fedora-setup.rst [hunk 2]
```diff
@@ -171,6 +176,10 @@ Fortran 编译器，其提供了 ``gfortran`` 命令::
 Intel 软件开发工具包
 ^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+
+   地震学新手可以先不安装此工具包，等日常科研中确实需要使用时再安装。
+
```
Comment (core-man) [inline source/computer/fedora-setup.rst:182]: I am thinking to move the installation into a dropdown box instead of adding a note only.

**Review by seisman — APPROVED** [review]

File: source/computer/fedora-setup.rst (hunk 2 above)
Comment (seisman) [reply source/computer/fedora-setup.rst:182]: Sounds good.

Comment (core-man) [reply source/computer/fedora-setup.rst]: Done in dba25ac.

Comment (seisman) [reply source/computer/fedora-setup.rst]: Do the same thing in the macOS guide?

Comment (core-man) [reply source/computer/fedora-setup.rst]: Done in dba25ac.

**Review by seisman — APPROVED** [review]

**Pull request merged**